pyproject.toml

```diff
@@ -0,0 +1,59 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "uncertainties"
+version = "3.1.7"
+authors = [
+    {name = "Eric O. LEBIGOT (EOL)", email = "eric.lebigot@normalesup.org"},
+]
+description = """\
+    Transparent calculations with uncertainties on the \
+    quantities involved (aka error propagation); \
+    fast calculation of derivatives').\
+    """
+readme = "README.rst"
+requires-python = ">=3.1"
```

Comment:
Because I think this change to pyproject.toml precludes easily installing the package with Python 2 (I don't think there is pyproject.toml support in the libraries left in Python 2 though maybe there are forks maintained somewhere with support for it), I set `requires-python>=3.1`. pip respects this value so I didn't want to set it too, so for example someone could install the package with Python 3.6 even though we only test 3.8+. Alternatively, we could just remove this line.